src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -81,6 +81,9 @@ def create_webapp(resource_group_name, name, plan, runtime=None, startup_file=No
         if not match['displayName'].startswith('node'):
             site_config.app_settings.append(NameValuePair("WEBSITE_NODE_DEFAULT_VERSION", "6.9.1"))
 
+    else:  # windows webapp without runtime specified
+        site_config.app_settings.append(NameValuePair("WEBSITE_NODE_DEFAULT_VERSION", "6.9.1"))
```

Comment:
Can you update the test to include this case as well?

Reply:
please make `6.9.1` as a constant and reuse with the same code 2 lines above

Reply:
fixed